language/move-lang/functional-tests/tests/libra_timestamp/basic.move

```diff
@@ -7,6 +7,8 @@ script {
         LibraTimestamp::initialize(account);
     }
 }
+// TODO(status_migration) remove duplicate check
```

Comment:
@vgao1996 Lots of weirdness going on here. Maybe I did something wrong with how I implemented adding the VMStatus to the function tests? (in addition to the transaction output)